src/lib/features/feature-search/feature-search-controller.ts

```diff
@@ -83,9 +83,7 @@ export default class FeatureSearchController extends Controller {
                 sortBy,
             } = req.query;
             const userId = req.user.id;
-            const normalizedTag = tag
-                ?.map((tag) => tag.split(':'))
-                .filter((tag) => tag.length === 2);
+            const normalizedTag = tag?.map((tag) => tag.split(':'));
```

Comment:
Now 1 length tags will also be used. 

**This is the question, should we split these already in controller, or do it in store.
If we do it here, we need to pass those in as 2 parameters. Full tags and half tags.
If we do it in store, we are leaking some normalizing logic in store.**

Reply:
I think I'm a little low on context: what's a 1-length tag? Like "simple:x"? what's a full tag and a half tag?

Reply:
Maybe it should happen in the service.

Reply:
Then we still need 2 parameters for tags in store method. Which may be fine, but seems a little much.

Reply:
@thomasheartman our tags consist of **type** and **value**. So a full tag would be the complete tag consisting of type and value. So `simple:mytag`, but previously we had possibility to search just by value or by type, without providing full tag definition. Just by writing simple, we got all `simple` tags.